content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: Un componente usando componentes de terceros
 order: 3
 domid: markdown-example
 ---
 
-React is flexible and provides hooks that allow you to interface with other libraries and frameworks. This example uses **remarkable**, an external Markdown library, to convert the `<textarea>`'s value in real time.
+React es flexible y provee medios que te permiten interactuar con otras librerías y marcos(frameworks). Este ejemplo usa **remarkable**, una librería externa de Markdown, que convierte los valores de `<textarea>` en tiempo real.
```

Comment:
1. `provee` doesn't sound right here. While it's a valid word, I think something in the lines of `proporciona` fits better.
2. Like I said before with the word "Plugin", "framework" is a widely understood concept that we lend from English and I've never seen it translated in Spanish on a context like this. So I just wouldn't translate `framework`.

Reply:
Although the use of «librería» as a translation for «library» is adopted by many people is also discouraged by others. I think [the position of the Royal Academy](https://mobile.twitter.com/RAEinforma/status/1090951164710010882) on this matter should be considered.
Also, «biblioteca» is the preferred term by many official docs, check out:
- https://developer.android.com/topic/libraries/support-library/?hl=es-419
- https://developers.facebook.com/docs/reference/php?locale=es_ES

Reply:
While I personally prefer "librería", I guess we shouldn't ignore the usage by official documentation from android and Facebook.